src/resources/app-en.json

```diff
@@ -802,8 +802,8 @@
   "noRespondentIdentifier": "No respondent identifier yet.",
   "noRespondents": "No Respondents yet.",
   "noParticipants": "No Participants yet.",
-  "noRespondentsForApplet": "No Respondents yet. Invite your first one by clicking the ‘Add Respondent’ button above.",
-  "noParticipantsForApplet": "No Participants yet. Invite your first one by clicking the ‘Add Participant’ button above.",
+  "noRespondentsForApplet": "This applet has no respondents",
```

Comment:
Please update the French version as well. 

_Cette applet n'a aucun répondant_ 

(via Google translate)